cypress/integration/mytools.js

```diff
@@ -1,150 +1,154 @@
+                })
+            // Make sure page is loaded first
+            cy.get('#tool-path').should('be.visible')
+            cy
+                .get('#register_tool_button')
+                .click()
+
+            cy
+                .get('#sourceCodeRepositoryInput')
+                .type("testnamespace/testname")
+                .wait(1000)
+
+            cy
+                .get('#imageRegistrySpinner')
+                .click()
+            cy
+                .contains('Amazon ECR')
+                .click()
+
+            cy
+                .get('#dockerRegistryPathInput')
+                .type('amazon.dkr.ecr.test.amazonaws.com')
+
+            cy
+                .get('#toolMaintainerEmailInput')
+                .type('test@email.com')
+
+            cy
+                .get('#imageRegistryInput')
+                .type('testnamespace/testname')
+
+            cy
+                .get('#submitButton')
+                .click()
+
+            cy
+                .get('#tool-path')
+                .should('contain', 'amazon.dkr.ecr.test.amazonaws.com/testnamespace/testname')
+
+            cy
+                .contains('Versions')
+                .click()
+
+            cy
+                .get('#addTagButton')
+                .click()
+
+            cy
+                .get('#versionTagInput')
+                .type('master')
+            cy
+                .get('#gitReferenceInput')
+                .type('master')
+
+            cy
+                .get('#addVersionTagButton')
+                .click()
+
+            cy
+                .get('#deregisterButton')
+                .click()
+            cy
+                .get('#deregisterConfirmButton')
+                .click()
+
+            // This should be activated later
```

Comment:
Maybe add what this is waiting for or link to a ticket

Reply:
Most of these lines here got formatted, didn't intentionally add this

Reply:
Still, no time like the present to figure this out